client/src/main/kotlin/app/cash/backfila/client/BackfilaManagementClient.kt

```diff
@@ -14,5 +16,9 @@ interface BackfilaManagementClient {
     scan_size: Long? = null,
     pkey_range_start: ByteString? = null,
     pkey_range_end: ByteString? = null,
-  )
+  ): CreateAndStartResponse?
+
+  fun checkStatus(
+    backfill_run_id: Long,
```

Comment:
I am really having second thoughts about this. If you need strong control over the execution and status I don't think Backfila is the solution you are looking for.

Reply:
Happy to remove things that are not needed. Wrote the protos in as a generally followed pattern; can be removed. The thing thats most relevant is being able to know which run ID was triggered which is a current gap. Not really looking for stronger control over anything on the backfila side

Reply:
The checkStatus is just a nice to have to be able to know if the job is running; no expectation change on how it works :)